third_party/move/move-compiler/tests/move_check_testsuite.rs

```diff
@@ -29,6 +29,9 @@ const SKIP_ATTRIBUTE_CHECKS_PATH: &str = "/skip_attribute_checks/";
 /// Root of tests which require to set warn_of_deprecation_use flag
 const WARN_DEPRECATION_PATH: &str = "/deprecated/";
 
+/// Root of unused functions tests warn_unused flag
+const WARN_UNUSED_PATH: &str = "/warn_unused/";
```

Comment:
Is this in prep for adding more tests? There does not seem to be any test that would fall under this path.

Reply:
Removed